app/reducers/user.js

```diff
@@ -31,7 +32,9 @@ const initialState = {
     playerLoginStatus: 0,
     originLinkStatus: 0,
     loginToken: null,
-    accountStorage: {},
+    accountStorage: {
+        [AccountStorageKeys.COMPACT_VIEW]: 'false',

```

Comment:
This will be ovewritten when VU hydrates the state, so it's unsafe to use it as a default.

Reply:
hmm, ok so i'll update expecting it not to be there at all.

Reply:
Removed default state. No other changes not needed to handle this key missing.